src/runtime/pkg/direct-volume/utils.go

```diff
@@ -0,0 +1,95 @@
+package volume
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+const (
+	mountInfoFileName        = "mountInfo.json"
+	defaultTimeout = 3 * time.Second
+)
+var kataDirectVolumeRootPath = "/run/kata-containers/shared/direct-volumes"
+
+// MountInfo contains the information for the Kata agent to mount a filesystem on the volume inside the guest VM.
+type MountInfo struct {
+	// The host path of the block device backing the volume.
+	DevicePath string `json:"device_path"`
```

Comment:
I think Device would be better than DevicePath. It needn't necessarily be a block device path on the host. It could, for example, be a URL?